portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.sps.utils.CommentDataStore;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles deleting comment content. */
+@WebServlet("/delete-comment")
+public class DeleteCommentServlet extends HttpServlet {
+
+    private CommentDataStore commentDataStore;
+
+    private static final String KIND = "Comment";
```

Comment:
Maybe bring this static constant to top

Reply:
Fixed

Reply:
Btw, do we need another KIND here if it's already in the CommentDataStore?

Reply:
Do u suggest that I directly get the KIND from CommentDatastore?

Reply:
Fixed